go/base/utils.go

```diff
@@ -65,16 +65,25 @@ func StringContainsAll(s string, substrings ...string) bool {
 }
 
 func ValidateConnection(db *gosql.DB, connectionConfig *mysql.ConnectionConfig) (string, error) {
-	query := `select @@global.port, @@global.version`
+	versionQuery := `select @@global.version`
 	var port, extraPort int
 	var version string
-	if err := db.QueryRow(query).Scan(&port, &version); err != nil {
+	if err := db.QueryRow(versionQuery).Scan(&version); err != nil {
```

Comment:
I don't see where `version` is used?

Reply:
The version is used in the function Inspector.validateConnection like before, this patch just prevent to get port from database.

Reply:
Thank you, you are right.